pkg/fsutils/utils.go

```diff
@@ -6,15 +6,6 @@ import (
 	"github.com/pkg/errors"
 )
 
-// AvailableBytesIn gets available capacity in bytes from file system of path.
-func AvailableBytesIn(path string) (uint64, error) {
-	stat, err := getDiskStats(path)
-	if err != nil {
-		return 0, err
-	}
-	return stat.Bavail * uint64(stat.Bsize), nil
-}
-
```

Comment:
Perhaps not used now, but does this have potential for future usage?

Reply:
In general, I agree with you.  But IMO this one probably doesn't have much use going forward.  It was used by the migrator to see if we could make a copy of RocksDB.